release/entrypoint.sh

```diff
@@ -57,15 +61,11 @@ DATA="
 }
 "
 
-if [ "$1" != "--dry-run" ]; then
-  # push the release content to github!
-  if ! curl --data "$DATA" https://api.github.com/repos/"$GITHUB_REPOSITORY"/releases?access_token="$GITHUB_TOKEN"; then
-    echo "\033[0;31mERROR: Unable to post github release tag information!\033[0m"  && exit 1
-  fi
-  echo -e "\n\033[1;32mA new github release tag has been created!\033[0m\n"
-else
-  echo -e "curl contents:\ncurl --data $DATA https://api.github.com/repos/$GITHUB_REPOSITORY/releases?access_token=$GITHUB_TOKEN"
+# push the release content to github!
+if ! curl --data "$DATA" https://api.github.com/repos/"$GITHUB_REPOSITORY"/releases?access_token="$GITHUB_TOKEN"; then
```

Comment:
It won't let me commend on the actual line, but we should use the commit shasum for the `target_commitish` here. This will prevent a race condition if someone does a release, then makes a commit to master right after.